lib/puppet/indirector/json.rb

```diff
@@ -67,10 +67,10 @@ def load_json_from_file(file, key)
   end
 
   def from_json(text)
-    model.convert_from('pson', text)
+    model.convert_from('json', text)
   end
 
   def to_json(object)
-    object.render('pson')
```

Comment:
We can probably lose the comment at the top of this file about being faster as pure ruby JSON since we'll primarily be using native extension JSON now.